src/index.html

```diff
@@ -2,11 +2,410 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <meta
+      name="viewport"
+      content="
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
+        minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet"
+    >
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header header--сontainer">
```

Comment:
In this case your outer styles (margin) are still related to `header` block, which is incorrect.

The correct way to add margins to the block is to make it the component of the higher-level block at the same time.
For example, add `class="page"` to your `<body>` element. Then you can safely add `page__header` class to the `<header>` element, and style the margins inside `page` block.

```
    <body class="page">
        <header class="page__header header">
            ...
```

Reply:
fixed